### PR TITLE
Integrations changes

### PR DESCRIPTION
**Pull request by MakMuftic**

### CODE REVIEW

File: cmd/hactar/commands/commands.go [hunk 1]
```diff
@@ -78,20 +78,30 @@ var StartCommand = &cli.Command{
...
-		})
-		if err != nil {
-			log.Error("Adding new node failed.", err)
-			return nil
-		} else if resp != nil && resp.StatusCode == http.StatusOK {
-			log.Info(fmt.Sprintf("New node added, url: %s address: %s", node.Node.Url, node.Node.Address))
+		// this check for existing nodes is just placeholder
+		nodes, _, err := hactarClient.Nodes.GetAllNodes()
+		if err == nil {
+			// TMP condition
```
Comment (morrigan) [inline cmd/hactar/commands/commands.go]: Elaborate please :) Backend should check if it already exists or not, so we should be good here

File: internal/stats/blocks.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func StartMonitoringBlocks(hactarClient *hactar.Client, lotusClient *lotus.Clien
 				return
 			case <-ticker.C:
 				log.Info("Block monitor ticked.")
-				SubmitNewBlockReport(hactarClient, lotusClient, currentSession)
+				// SubmitNewBlockReport(hactarClient, lotusClient, currentSession)
```
Comment (morrigan) [inline internal/stats/blocks.go]: Why commented out?

File: cmd/hactar/commands/commands.go (hunk 1 above)
Comment (MakMuftic) [reply cmd/hactar/commands/commands.go]: Fixed now, you were right. I had everything I need there :+1:

File: internal/stats/blocks.go (hunk 2 above)
Comment (MakMuftic) [reply internal/stats/blocks.go]: Backend didn't have that endpoint implemented so it was always failing while I tested integration (I made a hotfix :stuck_out_tongue: clearly). Just didn't remember to undo this action...

**Review by morrigan — APPROVED** [review]

**Pull request merged**